static/app/components/forms/controls/rangeSlider/index.tsx

```diff
@@ -202,6 +202,7 @@ function RangeSlider({
             value={sliderValue}
             aria-valuetext={labelText}
             aria-label={props['aria-label']}
+            hideLabel={!showLabel}
```

Comment:
could we also implement this with `formatLabel: () => null` ?

Reply:
I would prefer if we did what @TkDodo is mentioning here, but I would also ask about the use-case of why we need to hide the label in the first place?

Reply:
@TkDodo @JonasBa I actually did it this way initially, but it would still render like this (see the "empty" label rendering as a purple line):
![Screenshot 2025-04-29 at 9 39 20 AM](https://github.com/user-attachments/assets/0c6377fb-ba2c-488a-8d69-c3d74434b732)

if this is what we want i can change it to that but personally i thought the purple line was strange

Reply:
We need to change the internal implementation to not render the wrapper when the function returns false. Let me know if you need help with that.

Reply:
sgtm, i'll push the change rn!